### PR TITLE
:art: Add functionality to save card

### PR DESCRIPTION
**Pull request by LeahWinters**
Added functionality to the save card so when you click the close button, the card closes and the data is no longer stored in the local storage. Completed with Whole team, with Jeremy being navigator and Kyle chiming in to fix small bugs.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ saveForm.addEventListener('input', enableButton);
 window.addEventListener('load', retrieveLocalStorage);
 
 function clickHandlerOutfits() {
+  console.log(event.target.classList);
```
Comment (J-Poulter) [inline main.js:24]: This console log needs to be removed.

File: main.js [hunk 2]
```diff
@@ -31,6 +32,8 @@ function clickHandlerOutfits() {
     selectButton(buttonBackgrounds, backgrounds);
   } else if (event.target.classList.contains('save-button-js')) {
     saveOutfit();
+  } else if (event.target.classList.contains('far-js')) {
```
Comment (J-Poulter) [inline main.js:35]: We've decided to change this class name before merge so that it's more descriptive of what it is.

File: main.js [hunk 3]
```diff
@@ -104,3 +107,9 @@ function saveToLocalStorage(newOutfit) {
 function retrieveLocalStorage() {
   savedOutfitsSection.innerHTML = localStorage.getItem('cardSection');
 }
+
+function removeSaveCard() {
+  event.target.parentElement.remove();
+  saveToLocalStorage();
+  // parent.remove();
```
Comment (J-Poulter) [inline main.js:114]: Let's remove line 114 before push.

**Review by J-Poulter — CHANGES_REQUESTED** [review]
Hey I left some comments in the review. can you take a look and make changes before pushing.

File: main.js (hunk 1 above)
Comment (LeahWinters) [reply main.js:24]: Has been removed!

File: main.js (hunk 2 above)
Comment (LeahWinters) [reply main.js:35]: Has been changed to close-button-js

File: main.js (hunk 3 above)
Comment (LeahWinters) [reply main.js:114]: Has been removed!

**Comment by J-Poulter** [comment]
Ok looks good now!

**Pull request merged**